dev-python/docopt/docopt-0.6.2-r1.ebuild

```diff
@@ -4,7 +4,7 @@
 
 EAPI=5
 
-PYTHON_COMPAT=( python2_7 python3_{3,4,5} )
+PYTHON_COMPAT=( python2_7 python3_{3,4,5} pypy pypy3 )
```

Comment:
`grep -R "pypy pypy3" /usr/portage/dev-python/` shows that the right way of changing "pypy pypy3" into "pypy{,3}" is to write a script to do the task.